index.html

```diff
@@ -8,13 +8,104 @@
 
   <body>
     <h1>Tic Tac Toe</h1>
-    <div id="tic-tac-toe"></div>
+    <h3 id="game-status"></h3>
+    <div id="tic-tac-toe">
+    	<table>
+    		<tr id="rowOne">
+    			<td id="1"></td>
+    			<td id="2"></td>
+    			<td id="3"></td>
```

Comment:
Looks like the indentation went a little wonky here-- good thing to watch for to improve code readability